app/scripts/directives/nav.js

```diff
@@ -273,10 +291,47 @@ angular.module('openshiftConsole')
         };
 
         $scope.catalogLandingPageEnabled = !Constants.DISABLE_SERVICE_CATALOG_LANDING_PAGE;
-        var select = $elem.find('.selectpicker');
-        var options = [];
 
-        var updateOptions = function() {
+        var contextSelector = $elem.find('.contextselector');
+        var previousContextSelectorValue;
+        $scope.clusterConsoleURL = window.OPENSHIFT_CONSTANTS.TECTONIC_URL;
```

Comment:
Let's add an `OPENSHIFT_CONFIG` fallback for the short-term so that we can enable this without an extension.

```js
$scope.clusterConsoleURL = window.OPENSHIFT_CONSTANTS.TECTONIC_URL || window.OPENSHIFT_CONFIG.tectonicURL;
```